docs/ref/contrib/gis/geoquerysets.txt

```diff
@@ -109,6 +109,11 @@ MySQL       ``MBRContains(poly, geom)``
 SpatiaLite  ``Contains(poly, geom)``
 ==========  ============================
 
+.. versionadded:: 1.9
```

Comment:
Do you think we can remove these notes entirely when we removed the versionadded annotations in two versions? If not, it would be nice if they were "self-contained" as described in https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features.